docs/credentials-management/protecting-credentials.md

```diff
@@ -0,0 +1,67 @@
+# How to protect your credentials
+
```

Comment:
We need to write how the secrets are merged with the configmap so there won't be any confusion for the users.

Reply:
Basically we need to explicitly call out that the properties in the secret is appended to the properties in the configmap